advisor/src/main/kotlin/Advisor.kt

```diff
@@ -57,21 +57,17 @@ class Advisor(
         val ALL by lazy { LOADER.iterator().asSequence().toList() }
     }
 
-    fun retrieveVulnerabilityInformation(
-        ortResultFile: File,
-        skipExcluded: Boolean = false
-    ): OrtResult {
+    fun retrieveVulnerabilityInformation(ortFile: File, skipExcluded: Boolean = false): OrtResult {
```

Comment:
commit message: Did you mean "shorter fro**m**"?

Reply:
No, I just swapped words, fixed.